docs/tutorials/Canopy/soil_canopy_tutorial.jl

```diff
@@ -109,6 +110,67 @@ include(
 # Define the parameters for the soil model and provide them to the model 
 # parameters struct:
 
+# Heterotrophic respiration parameters
+P_sfc = FT(101e3)
+ν = FT(0.556)
+θ_a100 = FT(0.1816)
+D_ref = FT(1.39e-5)
+b = FT(4.547)
+D_liq = FT(3.17)
```

Comment:
Let's make a note of which parameters are constant in space and should be added to CLIMAParameters, and open an issue so that we keep track of this

Reply:
Do you mean constants?
I thought we were going to put all parameters in CLIMAParameters (both constants and variable parameters), and then specify what parameter we want to fit when we do data assimilation 
(which seemed weird to me - I like to be explicit about which parameters are constant or not). We can discuss more about this.

Reply:
yep! I mean any parameters that are the same everywhere in space (whether or not we calibrate them). Not super important right at this stage, but it might be good to open the issue while this is fresh.

Reply:
I will do that on the prognostic O2 PR - I hope we can merge this one before the sprint Friday